web/rootfs/defaults/settings-config.js

```diff
@@ -0,0 +1,259 @@
+{{ $XMPP_DOMAIN := .Env.XMPP_DOMAIN | default "jitsi-meet.example.com" -}}
```

Comment:
doesn't the extra dash `-}}` break things?

Reply:
Negatory, the dash simply removes whitespace after this statement, in this case the newline.  This keeps us from having a large number of newlines at the top of the file before the content.